package.json

```diff
@@ -49,14 +49,27 @@
     "lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
     "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "repo-tools test run --cmd mocha -- system-test/*.js --no-timeouts --bail",
-    "test": "repo-tools test run --cmd mocha -- test/*.js"
+    "system-test": "repo-tools test run --cmd mocha -- build/system-test/*.js --no-timeouts --bail",
+    "test": "repo-tools test run --cmd mocha -- build/test/*.js",
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
   },
   "dependencies": {
     "@google-cloud/logging": "^1.0.0",
-    "extend": "^3.0.1"
+    "@types/bunyan": "^1.8.3",
+    "extend": "^3.0.1",
+    "stream": "0.0.2",
```

Comment:
Was this on purpose?

Reply:
This may be added by accident. I can remove this line just fine now.